inception-project/inception
# #3155 - Upgrade RDF4J, Lucene, Solr, MTAS, etc.

**What's in the PR**
- Lucene 7.7.3 -> 8.11.1
- Solr 7.7.3 -> 8.11.1
- MTAS 7.7.1.0 -> 8.11.1.0
- Jena 3.17.0 -> 4.5.0
- RDF4J 3.7.7 -> 4.0.3

**How to test manually**
* Check that searching works in the annotation sidebar and in the KB on new and old projects
* Check that index rebuilding works for the annotation index and for KBs
* Check that bulk index rebuilding via the command line works (see upgrade notes in documentation)
* There has been no upgrade on ElasticSearch which also has transitive dependencies on Lucene/Solr, so it might be worth testing if the ElasitcSearch external repo connector still works

**Automatic testing**

**Documentation**
